content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,88 @@
 ---
 title: Getting Started
-# prettier-ignore
-cSpell:ignore: darwin dpkg GOARCH journalctl kubectl otelcorecol pprof tlsv zpages
+cSpell:ignore: docker dokey dpkg okey telemetrygen
 weight: 1
 ---
 
-If you aren't familiar with the deployment models, components, and repositories
-applicable to the OpenTelemetry Collector, first review the [Data Collection][]
-and [Deployment Methods][] page.
+The OpenTelemetry Collector listens to [traces](/docs/concepts/signals/traces/),
```

Comment:
```suggestion
The OpenTelemetry Collector receives [traces](/docs/concepts/signals/traces/),
```

I think that's the more accurate term since we have "receivers"?

Reply:
+1